spesifikasjoner/afpant/afpant-model/xsd/dsve-1.0.0.xsd

```diff
@@ -94,19 +102,53 @@
     <xs:sequence>
       <xs:element minOccurs="1" maxOccurs="1" name="avsender" type="Avsender"/>
       <xs:element minOccurs="1" maxOccurs="1" name="mottaker" type="Mottaker"/>
+      <xs:element minOccurs="1" maxOccurs="1" name="registerenheter" type="Registerenheter"/>
       <xs:element minOccurs="1" maxOccurs="1" name="restgjelddetaljer" type="Restgjelddetaljer"/>
       <xs:element minOccurs="1" maxOccurs="1" name="metadata" type="Metadata"/>
     </xs:sequence>
   </xs:complexType>
 
+  <xs:complexType name="Intensjonsforespoersel">
+    <xs:annotation>
+      <xs:documentation>
+        Forespørsel om metode for tinglysing
```

Comment:
Er det ikke egentlig intensjon fra bank, eller vil denne kunne benyttes motsatt vei?

Reply:
opprinnelig intensjon fra bank - men så var det dette scenarioet hvor megler endrer sin intensjon og må pushe endringen til bankene som tidligere har spurt (broadcast). for å unngå enda en meldingstype for dette scenarioet så tenkte jeg at meglersystemet kan gjenbruke "forespørsel om metode for tinglysing" for å kringkaste sin egen endrede intensjon.

det kan være at det er ryddigere å lage en separat meldingstype på dette (mer som en push-variant hvor man ikke forventer svar fra mottaker).

kanskje @javaguruen har noen innspill?

Reply:
Hadde ikke noen store tanker om dette, men enig at det er ryddig å ha en push type hvis man ikke forventer svar på denne.

Reply:
Har vi ikke her forvirret oss selv ved at det er benyttet samme navngivningsmetode på intensjon som for de andre meldingene der man faktisk sender en forespørsel og får et konkret resultat tilbake i form av feks en kjøpekontrakt eller en saldo?
Hva om vi lager kun èn intensjonsmeldingstype som kan brukes begge veier, underforstått man sender sin egen intensjon til en mottaker?
I så fall sender banken sin intensjon til megler ved en _intensjonsmelding_ og megler svarer deretter med en tilsvarende _intensjonsmelding_ tilbake til banken med sin intensjon?
Hvis megler senere må endre sin intensjon og dermed må informere alle banker som har sendt sin intensjon til megleren, så kan samme type _intensjonsmelding_ sendes til alle bankene. Så kan vi vurdere om bankene skal svare megleren med sin eventuelt oppdaterte intensjon.

Reply:
Vi innfører en ny meldingstype for 2-veis push:
  <xs:element name="intensjonsendring" type="Intensjonsendring"/>

Og renamer følgende: 
  <xs:element name="intensjonfrabank" type="IntensjonFraBank"/>
  <xs:element name="intensjonssvarframegler" type="IntensjonssvarFraMegler"/>

Alle som implementerer støtte for IntensjonFraBank eller IntensjonssvarFraMegler må også støtte mottak og sending av Intensjonsendring.

Bank melder inn følgende til AKELDO:
- Intensjonsendring, IntensjonssvarFraMegler

Megler melder inn følgende til AKELDO:
- Intensjonsendring, IntensjonFraBank